unlock-js/src/__tests__/v02/web3Service.test.js

```diff
@@ -379,8 +384,42 @@ describe('Web3Service', () => {
         web3Service._parseTransactionLogsFromReceipt(
           'hash',
           UnlockVersion.PublicLock,
-          receipt
+          receipt,
+          lockAddress
+        )
+      })
+
+      it('ignores events from outside our contract', async () => {
+        expect.assertions(1)
+        await versionedNockBeforeEach()
+        const EventInfo = new ethers.utils.Interface(erc20abi)
+        const receipt = {
+          blockNumber: 123,
+          logs: [
+            {
+              // fake ERC20 contract address
+              address: '0x1234123456789012345678901234567890567890',
```

Comment:
might be worthwhile to make this well named variable so that we can pull this comment